src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/components/TradeQuoteBase.tsx

```diff
@@ -0,0 +1,515 @@
+  const translate = useTranslate()
+  const estimatedTime = useMemo(() => {
+    if (quoteSteps.every(step => step.estimatedExecutionTimeMs === undefined)) return
+
+    return quoteSteps.reduce((acc, step) => {
+      return acc + (step.estimatedExecutionTimeMs ?? 0)
+    }, 0)
+  }, [quoteSteps])
+
+  return (
+    <TradeQuoteCardMetaItem
+      tooltip={translate('trade.quote.timeEstimate')}
+      isLoading={isLoading}
+      icon={TbClockHour3}
+      {...rest}
+    >
+      {estimatedTime !== undefined
+        ? estimatedTime === 0
+          ? '0s'
+          : prettyMilliseconds(estimatedTime)
+        : '-'}
+    </TradeQuoteCardMetaItem>
+  )
+}
+
+type TradeQuoteCardMetaPriceImpactProps = FlexProps & {
+  quote: TradeQuote | TradeRate | undefined
+  isLoading: boolean
+}
+export const TradeQuoteCardMetaPriceImpact: React.FC<TradeQuoteCardMetaPriceImpactProps> = ({
+  quote,
+  isLoading,
+  ...rest
+}) => {
+  const translate = useTranslate()
+
+  const {
+    isModeratePriceImpact,
+    isHighPriceImpact,
+    priceImpactPercentageAbsolute,
+    isPositivePriceImpact,
+  } = usePriceImpact(quote)
+
+  const priceImpactPc = useMemo(
+    () => priceImpactPercentageAbsolute?.div(100).toNumber(),
+    [priceImpactPercentageAbsolute],
+  )
+
+  const priceImpactColor = useMemo(() => {
+    switch (true) {
+      case isHighPriceImpact:
+        return 'text.error'
+      case isModeratePriceImpact:
+        return 'text.warning'
+      case isPositivePriceImpact:
+        return 'text.success'
+      default:
+        return undefined
+    }
+  }, [isHighPriceImpact, isModeratePriceImpact, isPositivePriceImpact])
+
+  return (
+    <TradeQuoteCardMetaItem
+      tooltip={translate('trade.quote.priceImpact')}
+      isLoading={isLoading}
+      icon={TbBolt}
+      {...rest}
+    >
+      <Amount.Percent value={priceImpactPc} color={priceImpactColor} />
+    </TradeQuoteCardMetaItem>
+  )
+}
+
+type TradeQuoteCardMetaItemProps = FlexProps & {
+  tooltip: string
+  icon: IconType
+  isLoading: boolean
+}
+export const TradeQuoteCardMetaItem: React.FC<TradeQuoteCardMetaItemProps> = ({
+  tooltip,
+  icon,
+  isLoading,
+  children,
+  ...rest
+}) => (
+  <Tooltip label={tooltip}>
+    <Flex gap={1} alignItems='center' {...rest}>
+      <Icon as={icon} color='text.subtle' boxSize={18} />
+      <Skeleton isLoaded={!isLoading}>{children}</Skeleton>
+    </Flex>
+  </Tooltip>
+)
+
+const WarningCircle: FC = () => (
+  <Circle size={6}>
+    <WarningIcon color='text.error' boxSize={4} />
+  </Circle>
+)
+
+type TradeQuoteCardErrorProps = {
+  hasAmountWithPositiveReceive: boolean
+  isAmountEntered: boolean
+  errors: ErrorWithMeta<TradeQuoteError>[]
+}
+export const TradeQuoteCardError: React.FC<TradeQuoteCardErrorProps> = ({
+  errors,
+  hasAmountWithPositiveReceive,
+  isAmountEntered,
+}) => {
+  const translate = useTranslate()
+
+  const error = errors?.[0]
+  const defaultError = { error: TradeQuoteValidationError.UnknownError }
```

Comment:
That looks incorrect, that *should* already be the default error:

https://github.com/shapeshift/web/blob/4747dad69903d96b31187cca438f8ee3e71304fc/src/state/apis/swapper/helpers/validateTradeQuote.ts#L121-L124

Although we should definitely use `.includes() like so:

https://github.com/shapeshift/web/blob/4747dad69903d96b31187cca438f8ee3e71304fc/src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx#L289-L292